### ISSUE TITLE
`check': undefined method `yes?

### ISSUE THREAD

**Issue by yegor256** [lpender/bummr#40]
This must be a bug?

```
$ bummr update
To run Bummr, you must:
- Be in the root path of a clean git branch off of master
- Have no commits or local changes
- Have a 'log' directory, where we can place logs
- Have your build configured to fail fast (recommended)
- Have locked any Gem version that you don't wish to update in your Gemfile
- It is recommended that you lock your versions of `ruby` and `rails in your Gemfile`
Your test command is: 'bundle exec rake'
Are you ready to use Bummr? (y/n) y
There is no log directory or you are not in the root
/Users/yegor/.rvm/gems/ruby-2.3.3/gems/bummr-0.2.0/lib/bummr/check.rb:17:in `check': undefined method `yes?' for #<Bummr::Check:0x007fb69644b8b8> (NoMethodError)
	from /Users/yegor/.rvm/gems/ruby-2.3.3/gems/bummr-0.2.0/lib/bummr/cli.rb:10:in `check'
	from /Users/yegor/.rvm/gems/ruby-2.3.3/gems/bummr-0.2.0/lib/bummr/cli.rb:18:in `update'
	from /Users/yegor/.rvm/gems/ruby-2.3.3/gems/thor-0.19.4/lib/thor/command.rb:27:in `run'
	from /Users/yegor/.rvm/gems/ruby-2.3.3/gems/thor-0.19.4/lib/thor/invocation.rb:126:in `invoke_command'
	from /Users/yegor/.rvm/gems/ruby-2.3.3/gems/thor-0.19.4/lib/thor.rb:369:in `dispatch'
	from /Users/yegor/.rvm/gems/ruby-2.3.3/gems/thor-0.19.4/lib/thor/base.rb:444:in `start'
	from /Users/yegor/.rvm/gems/ruby-2.3.3/gems/bummr-0.2.0/bin/bummr:4:in `<top (required)>'
	from /Users/yegor/.rvm/gems/ruby-2.3.3/bin/bummr:22:in `load'
	from /Users/yegor/.rvm/gems/ruby-2.3.3/bin/bummr:22:in `<main>'
	from /Users/yegor/.rvm/gems/ruby-2.3.3/bin/ruby_executable_hooks:15:in `eval'
	from /Users/yegor/.rvm/gems/ruby-2.3.3/bin/ruby_executable_hooks:15:in `<main>
```

**Comment by lpender** [comment]
Strange that method comes with Thor, which is implemented by the CLI class:

http://www.rubydoc.info/github/wycats/thor/Thor%2FShell%2FBasic:yes%3F

**Comment by lpender** [comment]
I noticed that the class doesn't explicitly inherit from Thor actually.

Just released `v0.1.7`-- please check and see if that works for you.

**Comment by yegor256** [comment]
@lpender how do you want me to try? And what is `v0.1.7`? What did you release? bummr? I'm lost :)

**Comment by lpender** [comment]
@yegor256 Yes, I released `v0.1.7` of Bummr :) Please update `gem update bummr` and try again.

**Comment by yegor256** [comment]
@lpender thanks, that bug is gone
